### DISCUSSION TITLE
PnP Search result - details list - group by on category and subcategory (double groupBy)

### DISCUSSION THREAD

**Discussion by MelanieO365** [microsoft-search/pnp-modern-search#3604] [Ideas]
It would be nice to make it possible to add 2 layers of group by for the details list layout.
Like it is possible in views of lists and libraries.
For example group by on Category and then group by on sub category.

**Comment by kasperbolarsen** [comment]
I agree, and we would welcome a PR to this effect from you or your organization :-)
The new layout would fit perfectly in the Layouts repository here, https://github.com/microsoft-search/pnp-modern-search-layouts/

**Comment by Mr-Matthieu** [comment]
@MelanieO365 Any news on your idea :)?

**Comment by patrikhellgren** [comment]
This feature will be part of the next release. See PR #3707. You can even add more levels than two if you want to.
